src/libraries/System.Private.Xml/src/System/Xml/Serialization/Models.cs

```diff
@@ -90,16 +94,20 @@ internal ArrayModel GetArrayModel(Type type)
     internal abstract class TypeModel
     {
         private readonly TypeDesc _typeDesc;
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
         private readonly Type _type;
         private readonly ModelScope _scope;
 
-        protected TypeModel(Type type, TypeDesc typeDesc, ModelScope scope)
+        protected TypeModel(
+            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
```

Comment:
(nit) this formatting is a little hard to read. I've usually been following the pattern of one param per line, and the attribute on the same line as the param it annoates:

```C#
protected TypeModel(
            [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)] Type type,
            TypeDesc typeDesc, 
            ModelScope scope)
```